### PR TITLE
Update/jsoniter scala core 2.13.13

### PR DESCRIPTION
**Pull request by daddykotex**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -634,14 +639,22 @@ val isCE3 = settingKey[Boolean]("Is the current build using CE3?")
 
 lazy val Dependencies = new {
 
+  object ScalaJS {
+    val scalaJsJavaSecureRandom: Def.Initialize[ModuleID] = Def.setting(
+      ("org.scala-js" %%% "scalajs-java-securerandom" % "1.0.0").cross(
+        CrossVersion.for3Use2_13
```
Comment (daddykotex) [inline build.sbt:645]: I'm not sure if we should add this (I remember seeing that using `for3Use2_13` was not appropriate for libraries)

Comment (armanbilge) [reply build.sbt:645]: I recommend using `cats.effect.std.UUIDGen` and avoiding this dep altogether. See:
- https://github.com/typelevel/cats-effect/pull/2945

Comment (daddykotex) [reply build.sbt:645]: I will attempt to rewrite, but we depend on cats-effect transitively so I'll have to update the build a bit. In the mean time, FYI @Baccata :

Screen Shot 2022-04-13 at 11 17 15

Comment (daddykotex) [reply build.sbt]: my latest changes should do it!

File: modules/tests/src-ce2/UUIDGen.scala [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+ *     https://disneystreaming.github.io/TOST-1.0.txt
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package cats.effect.std
```
Comment (Baccata) [inline modules/tests/src-ce2/UUIDGen.scala:17]: hehehe nice 😄

**Review by Baccata — APPROVED** [review]
Approved, as long as the build is green. 

Good solution :)

File: modules/tests/src-ce2/UUIDGen.scala (hunk 2 above)
Comment (daddykotex) [reply modules/tests/src-ce2/UUIDGen.scala:17]: fake it till you can make it

**Pull request merged**